components/Pagination/Pagination.js

```diff
@@ -14,88 +14,97 @@ class Pagination extends React.Component {
     super(defaultsDeep(props, Pagination.defaultProps));
   }
 
-  previousPageLink(pager) {
-    if (pager.isFirstPage()) {
-      return <PaginationHiddenLink label={this.props.labels.prev} />;
+  handleClick(pageNumber, event) {
+    event.preventDefault();
+    this.props.setCurrentPage(pageNumber);
+  }
+
+  pageLink({label, ariaLabel, pageNumber, className = null, isDisabled = false, isActive = false}) {
+    var handleClick = this.handleClick.bind(this, pageNumber);
+
+    className = cx(bem('item'), className);
+    if (isDisabled) {
+      className = cx(bem('item', 'disabled'), this.props.cssClasses.disabled, className);
+    }
+    if (isActive) {
+      className = cx(bem('item-page', 'active'), this.props.cssClasses.active, className);
     }
 
+
     return (
-      <PaginationLink
```

Comment:
Because it was only passing the props to a `li > a`, all logic was already handled in the `Pagination` component. I thought this was one layer too many. I don't see the value of it, could you explain?